frontend/src/actions/productActions.js

```diff
@@ -15,24 +15,41 @@ import {
   PRODUCT_DELETE_REQUEST,
   PRODUCT_DELETE_FAIL,
   PRODUCT_DELETE_SUCCESS,
+  PRODUCT_CATEGORY_LIST_SUCCESS,
+  PRODUCT_CATEGORY_LIST_REQUEST,
+  PRODUCT_CATEGORY_LIST_FAIL,
 } from '../constants/productConstants';
 
-export const listProducts = ({ seller = '', name = '' }) => async (
-  dispatch
-) => {
+export const listProducts = ({
+  seller = '',
+  name = '',
+  category = '',
```

Comment:
I'm getting a : TypeError: Cannot read property 'category' of undefined

Reply:
I am also facing the same error